core/p2p/p2p_disabled.go

```diff
@@ -19,3 +19,11 @@ func LLamaCPPRPCServerDiscoverer(ctx context.Context, token string) error {
 func BindLLamaCPPWorker(ctx context.Context, host, port, token string) error {
 	return fmt.Errorf("not implemented")
 }
+
+func GetAvailableNodes() []NodeData {
```

Comment:
@mudler if you have a sec, do you mind explaining why `p2p_disabled.go` is... useful? Mostly for my own understanding here - I'd have either just checked if p2p was enabled when `GetAvailableNodes` was called, or if performance was to be optimized, dump both versions in the same file and select the mock vs the full impl based on that setting. I'm probably missing something interesting here 😄

Reply:
it's actually because it is behind the `GO_TAGS` user flag. If `GO_TAGS` contains p2p then the `p2p.go` file is built, otherwise the other one.

Reply:
Got it,  thanks for pointing out this is a compile time option vs a config setting